docker/compose/dev/install-configs.sh

```diff
@@ -15,18 +14,27 @@ if [[ ! $demo_host_address =~ ^http ]]; then
     demo_host_address="http://${demo_host_address}"
```

Comment:
This is too naive. What if the env has a https:// address. This is the address which will be externally communicated. So the host address should simply be provided with http(s) and we should throw an error if that is not done

Reply:
outdated